tests/api/test_grid_workflow.py

```diff
@@ -124,3 +124,48 @@ def search_eager(lat=None, lon=None, **kwargs):
     assert len(padded_tile) == 1
     assert padded_tile[1, -2, ti].shape == (1, 14, 14)
     assert len(padded_tile[1, -2, ti].sources.values[0]) == 2
+
+
+def test_cell():
+    """ Test GridWorkflow with padding option. """
```

Comment:
Just trying to work out the difference to the above function. I can see that this tests `list_cells()`, which the above function doesn't, but other than that it's not clear what's going.

Can you add some comments to explain a little better the purpose of the test please.